README-ES.md

```diff
@@ -130,31 +132,31 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
     * [RxJS](http://reactivex.io/)
     * [ImmutableJS](https://facebook.github.io/immutable-js/)
     * [Ramda](https://ramdajs.com/)
-12. Pruebas
-    1. Pruebas Unitarias
+12. Pruebas (Testing)
+    1. Pruebas Unitarias (Unit Testing)
         * [Jest](https://facebook.github.io/jest/)
         * [Enzyme](http://airbnb.io/enzyme/)
         * [Sinon](http://sinonjs.org/)
         * [Mocha](https://mochajs.org/)
         * [Chai](http://www.chaijs.com/)
         * [AVA](https://github.com/avajs/ava)
         * [Tape](https://github.com/substack/tape)
-    2. Pruebas de extremo a extremo
+    2. Pruebas de extremo a extremo (End To End Testing, E2E)
         * [Selenium](https://www.seleniumhq.org/), [Webdriver](http://webdriver.io/)
         * [Cypress](https://cypress.io/)
         * [Puppeteer](https://pptr.dev/)
         * [Cucumber.js](https://github.com/cucumber/cucumber-js)
         * [Nightwatch.js](http://nightwatchjs.org/)
-    3. Pruebas de integración
+    3. Pruebas de Integración (Integration Tests)
         * [Karma](https://karma-runner.github.io/)
-13. Internacionalización
+13. Internacionalización (Int, i18)
```

Comment:
```suggestion
13. Internacionalización
```

Or 

```suggestion
13. Internacionalización (i18n)
```

`Int` may convey a data type for the reader and not Internationalization.